src/libostree/ostree-core.c

```diff
@@ -183,6 +184,35 @@ ostree_validate_rev (const char *rev,
 }
 
 /**
+ * ostree_validate_remote_name:
+ * @remote_name: A remote name
+ * @error: Error
+ *
+ * Returns: %TRUE if @remote_name is a valid remote name
+ * Since: 2017.7
```

Comment:
This one is `Since 2017.8` though.

But yeah, since this patch is nicely isolated, would you mind splitting it to a separate PR?